### ISSUE TITLE
ospos on local network

### ISSUE THREAD

**Issue by mrwebinn** [jekkos/opensourcepos#308]
Hi there ever one is some one please help how can I use OSPOS on my local network having only one server opreting the OSPOS software and other computers just connect with them

**Comment by arionkosturi** [comment]
If you're using Windows, install wamp. Make it autorun on start up. Install OSPOS. Access the POS through the server IP.

**Comment by odiea** [comment]
You will need to find your server ip then in a browser from a different computer enter the server ip address/opensourcepos or what ever you named it when you installed it.

**Comment by mrwebinn** [comment]
Thanks for your great replay 
I had install wamp and my ospos is currently running 
How can I find my SERVER IP address of OponSourcePos on my server and how can i run them on other computers ??
One more thing Does I need to install wamp also on other computers or not (( I think no need to install WAMP on other computers its just mine idea ))

**Comment by daN4cat** [comment]
You need one server (wamp) with OSPOS running on it and then you can access it from different clients running web browsers.

To find your ip on the server, open a Command Prompt and then type ipconfig. With that command you should find the ip address of your server. The go to your webbrowser and type that IP in it.

**Comment by mrwebinn** [comment]
Thanks alot network configuration done thanks
How does I connect it with any website
Can I store data in both online and of line same data

**Comment by daN4cat** [comment]
There is no offline facility, which means nothing of the the data is stored on the client with web browser. You need to have a connection to your web server running OSPOS.

By the way, I would suggest to make up your mind about web services from a generic point of view before playing with OSPOS further. I sense that you need to understand a more general part before playing with something in specific.

**Comment by mrwebinn** [comment]
I think maybe I could not write the question in right direction 
By the means of 
  ..Offline >> Data store on wamp server on my local Sever Machine and 
  ..Online means On the website database server
And Bro one more thing need to know which file store all my data information ??
Thanks

**Comment by daN4cat** [comment]
OSPOS data is stored on the MySQL server you point in your OSPOS config.

Beside that, all the rest is a non-OSPOS question and you can find answers in forum talking about how to replicate MySQL data or make it redundant between 2 servers. Stuff that I'm afraid I won't personally know without looking in one of those forum myself.

**Comment by maravento** [comment]
Someone could indicate the link to the instructions of the program where it appears clearly "how a client can access the application (server in local network) from the browser" (I have searched and it does not appear anywhere)

**Comment by odiea** [comment]
Open the browser then input the address/ followed by whatever you named this/ then public and it should work.

**Comment by sirajomusa19** [comment]
I have installed OSPOS on my local machine. The installation is working well. However, after some time, it turns out to be unacceptably slow. I followed some suggestions and as a result changed the host file on windows to disable ipv6, I also commented "LoadModule cgi_module modules/mod_cgi.so"  in httpd.conf.  At first, it improved, but after a day, it seemed to have gone back to the earlier state of SLOW.
